Cargo.toml

```diff
@@ -4,6 +4,10 @@ debug = true
 split-debuginfo = "packed"
 lto = false                # Preserve the 'thin local LTO' for this build.
 
+[profile.release]
+split-debuginfo = "unpacked"
+lto = "thin"
```

Comment:
I'm not saying we should or should not have these. My point was the previous PR didn't do what it said it did, it changed other stuff, which is confusing for our history.

How recent is `profile.release`? Do we actually want to set these defaults for operators?

Reply:
If this is purely for benchmarks, then we should use the bench profile: `profile.bench` not `release`